bpf/process/generic_calls.h

```diff
@@ -413,6 +419,7 @@ FUNC_INLINE void extract_arg(struct event_config *config, int index, unsigned lo
 		struct extract_arg_data extract_data = {
 			.btf_config = btf_config,
 			.arg = a,
+			.null_ptr_found = null_ptr_found,
```

Comment:
I think the field should probably be called `err` because you catch the error from `probe_read`